<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glaxnimate::gui::ShapeToolWidget</class>
 <widget class="QWidget" name="ShapeToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>194</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QGroupBox" name="group_box">
     <property name="title">
      <string>Create</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="check_raw_shape">
        <property name="text">
         <string>Raw Shape</string>
        </property>
        <property name="icon">
         <iconset theme="shapes">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="check_group">
        <property name="text">
         <string>Group</string>
        </property>
        <property name="icon">
         <iconset theme="object-group">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="check_layer">
        <property name="text">
         <string>Layer</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-layers">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_fill">
        <property name="text">
         <string>Fill</string>
        </property>
        <property name="icon">
         <iconset theme="format-fill-color">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_stroke">
        <property name="text">
         <string>Stroke</string>
        </property>
        <property name="icon">
         <iconset theme="format-stroke-color">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_fill</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_stroke</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_layer</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_group</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_raw_shape</sender>
   <signal>clicked()</signal>
   <receiver>ShapeToolWidget</receiver>
   <slot>check_checks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check_checks()</slot>
 </slots>
</ui>
